tests/models/neural_networks/test_rnn.py

```diff
@@ -43,11 +43,10 @@ def mocktrain(self):
         model.train()
         model.save_model()
 
-        assert (tmp_path / 'models/one_month_forecast/rnn/model.pkl').exists(), \
+        assert (tmp_path / 'models/one_month_forecast/rnn/model.pt').exists(), \
```

Comment:
why are we changing to the pytorch save data file? is this a general fix or specific for GPUs?

Reply:
It's to ensure we can load it onto the correct device:
https://pytorch.org/tutorials/beginner/saving_loading_models.html#saving-loading-model-across-devices